rfcs/text/0000-tech-planning-process.md

```diff
@@ -0,0 +1,134 @@
+“grassroots” knowledge about the technical requirements of the DCP on a quarterly basis to guide the 
+[quarterly Release process](https://github.com/HumanCellAtlas/dcp-community/blob/4bb18c65af5b244460cf404465b1dbc4ee0cc65e/rfcs/text/0000-roadmaps%2Bplanning.md#planning-and-executing-a-release)
+via a Technical Objectives RFC. This document details the contents of the Technical Objectives RFC as well as the
+process that must occur such that this artifact is drafted and curated according to a timeline that will allow us to
+present the artifact to the quarterly TL+PM objective curation meeting.
+
+This proposed TL Quarterly Planning Process is **owned by the collection of Tech Leads in the DCP** and is intended to
+be a live process that iterates over time as required. The first iteration of the planning process is detailed below.  
+
+Note: The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT
+RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14, RFC2119, and RFC8174
+when, and only when, they appear in all capitals, as shown here.
+
+### User Stories
+
+As a Tech Lead of the DCP, I am able to surface important technical priorities to Product Management that will be
+incorporated into the yearly roadmap. 
+
+As a developer on the DCP, I am able to allocate enough time to work on high priority technical architectural issues
+knowing that the work will facilitate quicker development of feature requests in the long term as informed by
+the roadmap themes.
+
+## Definitions
+**Quarter:** One quarter is equal to 13 weeks. Generally the quarters have the following timeline:
+
+- Quarter 1 (Q1): first Monday of January -> last Friday of March
+- Quarter 2 (Q2): first Monday of April -> last Friday of June
+- Quarter 3 (Q3): first Monday of July -> last Friday of September
+- Quarter 4 (Q4): first Monday of October -> last Friday of December (before the holidays)
+
+**Milestone:** a four week timeline/duration in a Release (sometimes five when a holiday is involved).
+
+**Release:** composed of three Milestones and occur on a quarterly cadence.
+
+**TL Roadmapping Shepherd:** Each quarter one TL acts as the “Shepherd” to make sure the TL planning process takes place
+in time to produce the expected Technical Objectives RFC for presentation to the TL+PM planning meeting. The Shepherd
+role will rotate once a quarter according to a spreadsheet drawn up post the approval of this RFC. 
+
+**Technical Objectives RFC:** The final artifact that will be presented to the TL+PM planning call as a representation
+of what TL would like to prioritize work on in the following quarter.
+
+**Component:** Also known as a “Box”; represents a piece of the DCP led by a TL and worked on by a team of software
+'engineers. Today, the components are: Upload, Ingest, Data Store Service (DSS), Azul/Data Browser, Secondary Analysis,
```

Comment:
```suggestion
engineers. Today, the components are: Upload, Ingest, Data Store Service (DSS), Azul/Data Browser, Secondary Analysis,
```

Reply:
I suspect @jkaneria and @kbergin might want Secondary Analysis separated out per their charters.

Reply:
I would suggest using _chartered components_ instead of listing specific components which will be outdated as soon as Stanford creates their charter.

`[chartered components](https://github.com/HumanCellAtlas/dcp-community/tree/master/charters)`

Reply:
Great suggestion! Modified!